src/WorkItemFiling/WorkItemFilingUtilities.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.WorkItemFiling
+{
+    public static class WorkItemFilingUtilities
+    {
+        internal const string PROJECT = "account";
+        internal const string ACCOUNT = "project";
+
+        private static readonly string GitHubUriPattern = $"^https://github\\.com/(?<{ACCOUNT}>[^/]+)/(?<{PROJECT}>[^/]+)$";
```

Comment:
>PROJECT [](start = 106, length = 7)

Note: as per previous PR I've added named groups so that we can use the RegexOptions.ExplicitCapture option (which improves performance).